sky/check.py

```diff
@@ -75,14 +99,21 @@ def check_one_cloud(cloud_tuple: Tuple[str, clouds.Cloud]) -> None:
 
         # Pretty print for UX.
         if not quiet:
+            if len(clouds_set) == 0:
+                enabled_clouds_for_output = enabled_clouds
+            else:
+                enabled_clouds_for_output = [
+                    cloud for cloud in enabled_clouds
+                    if cloud.lower() in clouds_set
+                ]
```

Comment:
I think we should not have this handling to show only the clouds that were enabled in the current `sky check` run, and instead show _all_ clouds that are currently enabled. E.g., it can be confusing to see a blank `Enabled clouds` after running `sky check <cloud>` even though other clouds are still enabled.

```
(base) ➜  ~ sky check kubernetes
Checking credentials to enable clouds for SkyPilot.
  Kubernetes: disabled
    Reason: ValueError: Failed to load Kubernetes configuration. Please check if your kubeconfig file is valid.
To disable Kubernetes for SkyPilot: run `sky check`.

To enable a cloud, follow the hints above and rerun: sky check
If any problems remain, refer to detailed docs at: https://skypilot.readthedocs.io/en/latest/getting-started/installation.html

🎉 Enabled clouds 🎉
(base) ➜  ~
```

In other words, the final `🎉 Enabled clouds 🎉` should show the current state of SkyPilot with all clouds currently enabled, not just the changes made in this `sky check` call. wdyt?

Reply:
Fixed! Agreed - I had only given it this behavior before since I think one of the prior comments showed an example of just showing whether the specified clouds were enabled or not